toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/team/TeamDaoService.java

```diff
@@ -0,0 +1,38 @@
+package io.toast.tk.dao.service.dao.access.team;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.github.jmkgreen.morphia.query.Query;
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.name.Named;
+
+import io.toast.tk.dao.domain.impl.common.IServiceFactory;
+import io.toast.tk.dao.domain.impl.team.TeamImpl;
+import io.toast.tk.dao.domain.impl.team.UserImpl;
+import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
+import io.toast.tk.dao.service.dao.common.CommonMongoDaoService;
+import io.toast.tk.dao.service.init.DbStarter;
+
+public class TeamDaoService extends AbstractMongoDaoService<TeamImpl> {
+
+	public interface Factory extends IServiceFactory<TeamDaoService> {
+	}
+
+	@Inject
+	public TeamDaoService(final DbStarter starter, 
+			final CommonMongoDaoService commonService,
+			@Named("default_db") final String defaultDb, 
+			@Assisted final String databaseName,
+			final TeamDaoService.Factory tDaoServiceFactory) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "tDaoServiceFactory". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1172)